sos/plugins/lustre.py

```diff
@@ -0,0 +1,44 @@
+### This program is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+
+from sos.plugins import Plugin, RedHatPlugin
+
+class Lustre(Plugin, RedHatPlugin):
+    '''Lustre filesystem'''
+
+    plugin_name = 'lustre'
+    profiles = ('storage','network','cluster',)
+    packages = ('lustre', 'lustre-client',)
+
+    def setup(self):
+        self.add_cmd_output("lctl debug_kernel")
+        self.add_cmd_output("lctl device_list")
+
```

Comment:
You can do these as a single `add_cmd_output()` call by passing a list of strings, something like:

```python
    self.add_cmd_output([
        "lctl debug_kernel",
        "lctl device_list",
    ])
```

The call overhead isn't an issue - it just looks slightly neater.